supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/request/GetRequestHandler.scala

```diff
@@ -78,7 +80,7 @@ trait GetRequestHandler { this: RequestHandler =>
       }
     } catch {
       case npe: NullPointerException =>
-        throw new RuntimeException(s"Caught NPE while processing operation ${operation.getName}: $operation", npe)
+        throw new RuntimeException(s"Caught NPE while interpreting operation ${operation.getName}: ${ExceptionUtils.getStackTrace(npe)} ")
```

Comment:
Maybe I'm missing something but you could just pass in the npe to the runtime exception:

```scala 
        throw new RuntimeException(s"Caught NPE while interpreting operation ${operation.getName}", npe)
```

Reply:
That is what the old code did and the result was not satisfactory: no stack trace was being printed

Example output: https://travis-ci.com/broadinstitute/cromwell/jobs/187652847

Reply:
I see.  I think that is an unfortunate logging situation that is unrelated to this particular issue.